### DISCUSSION TITLE
Sunset/sunrise node

### DISCUSSION THREAD

**Discussion by Dirnei** [Dirnei/node-red-contrib-zigbee2mqtt-devices#68] [Ideas]
Original Issue #41

Copied text:

Would it make sense to add a node for sunrise/sundown or in general to get insight if the sun is shining or not?
It's strictly speaking not really part of this lib. And there might be libs that do that better.
I have not researched this yet, just wanted to note the thought.

**Comment by Dirnei** [comment]
@mdhom just sent me a cool link: https://flows.nodered.org/node/node-red-contrib-cron-plus

**Reply by Andreas-Huber** [reply]
Ah, you closed the issue. Yeah I was pretty sure that something like this exists.
For sure this solves the issue and it does not make sense to add that to the lib if such an extensive solution already exists.
